CHANGELOG.md

```diff
@@ -5,32 +5,35 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 ## [Unreleased]
 
+### Changed
+- Fix for deprecated "io/ioutil" in [#39](https://github.com/cybozu-go/log/pull/39)
+
```

Comment:
Do you not list the update dependencies in the Changelog?
I commented because it is written in the Changelog of other repositories and I was anxious about it.

Reply:
@zeroalphat 
I hesitated to add "update dependencies" to the list because I did not update the package dependencies for this repository.
Furthermore this repository does not depend on any other packages.

I did modify the GitHub Actions workflow indeed, but the modification is just for testing.
IMO we don't need to list the modification of the testing environment as an independent item in CHANGELOG.

Reply:
Thanks for the explanation, I understand.